### PR TITLE
62 fix bugs books route

### PR DESCRIPTION
**Pull request by Saeed99Madi**

### CODE REVIEW

File: src/controllers/index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 const path = require('path');
 
 const homePage = (req,res) => {
-    if(req.user)
```
Comment (Saeed99Madi) [inline src/controllers/index.js:4]: req.user !!!!!

File: src/middleware/isAuthenticated.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
 const jwt = require('jsonwebtoken');
-
+require('dotenv').config()
 const { SECRET_KEY } = process.env;
 
 const userAuthenticated = (req, res, next) => {
-  const { token } = req.cookies.token;
```
Comment (Saeed99Madi) [inline src/middleware/isAuthenticated.js:6]: dose not need wrap in object

File: src/middleware/isAuthenticated.js [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
     res.status(400).send('Access Denied');
   } else {
     jwt.verify(token, SECRET_KEY, (err, encoded) => {
-      if (!err) {
-        res.status(400).send('Invalid Token');
+      if (err) {
+        console.log(err);
+        next(err)
       } else {
-        res.cookie('token', encoded);
```
Comment (Saeed99Madi) [inline src/middleware/isAuthenticated.js:14]: why ` res.cookie('token', encoded) ` !!!

File: src/routes/users.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
 const router = require('express').Router();
 const userConroller = require("../controllers/users/index");
 const homePage = require('../controllers/index');
-const userAuthenticated = require('../middleware/index')
```
Comment (Saeed99Madi) [inline src/routes/users.js:4]: needs wrap

File: src/routes/users.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
+const {userAuthenticated} = require('../middleware/index')
 
-router.get('/home', userAuthenticated, homePage);
```
Comment (Saeed99Madi) [inline src/routes/users.js:6]: make sure to use same endpoint in frontend dom == > in script.js file

**Review by Ibtisam-Hemmo — APPROVED** [review]

**Pull request merged**